domains/yutong.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "YutongHanOfficial",
+    "email": "yutongbusiness@gmail.com"
+  }
+  {
+  "record": {
+    "TXT": "replit-verify=e5ac2ad9-5903-4c72-9092-7da12688a4c2"
```

Comment:
Not the right way to connect is-a.dev domain to replit, please read the docs https://www.is-a.dev/docs/replit/